### DISCUSSION TITLE
Multiple coin node daemons

### DISCUSSION THREAD

**Discussion by snoby** [coinfoundry/miningcore#1508] [General]
We are running multiple geo region stratums currently and I'm interested in making the system more resilient.  In the config file there is a section for multiple node daemons to be setup.

For low latency and short block time coins I assume each stratum has a node instance daemon sitting next to the stratum instance. These nodes can just be there to feed the stratum the next block and of course do not need the wallet setup on them.

Now the master instance of mc needs to have the coin node instance with the wallet next to the master, but my question is, for redundancy if we run two coin node instances, they must be mirrors of each other with the same wallet installed correct?

**Comment by oliverw** [comment]
Support for multiple daemons was dropped a while ago because it was never fully implemented.

All of your other assumptions are correct.
